EIPS/eip-6821.md

```diff
@@ -0,0 +1,41 @@
+---
+eip: 6821
+title: Support ENS Name for Web3 URL
+description: A mapping from an ENS name to the contract address in Web3 URL
+author: Qi Zhou (@qizhou), Qiang Zhu (@qzhodl)
+discussions-to: https://ethereum-magicians.org/t/eip-6821-support-ens-name-for-web3-url/13654
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-04-02
+requires: 137, 4804
```

Comment:
Should this standard reference https://eips.ethereum.org/EIPS/eip-634?

Reply:
Added.  One question is that EIP-634 is in `stagnant` status, which prevents finalizing the standard.  Do you have any thoughts on that?

Reply:
It's kinda unfortunate. Might be worth opening a PR to bring 634 out of stagnant.

Reply:
https://github.com/ethereum/EIPs/pull/6841